ecs-cli/modules/cli/local/up_app.go

```diff
@@ -40,26 +43,50 @@ import (
 // If the container is not running, this command creates a new network for all local ECS tasks to join
 // and communicate with the Amazon ECS Local Endpoints container.
 func Up(c *cli.Context) {
-	// TODO When we don't provide any flags, Create() should just check if a "./docker-compose.local.yml" already exists.
-	// If so then Create() should do nothing else, otherwise it should error.
-	Create(c)
+	if err := options.ValidateCombinations(c); err != nil {
+		logrus.Fatalf(err.Error())
+	}
+	composePath, err := createComposeFile(c)
+	if err != nil {
+		logrus.Fatalf("Failed to get compose file path due to:\n%v", err)
```

Comment:
Nitpick: error message should not be capitalized

Reply:
Ah yeah but this is a log statement to the user so I think it's fine to capitalize.

If we created a `errors.New` then I'd agree :)